tools/profiles/release.json

```diff
@@ -14,6 +14,13 @@
                "-Wl,--wrap,_calloc_r", "-Wl,--wrap,exit", "-Wl,--wrap,atexit",
                "-Wl,-n"]
     },
+    "ARMC6": {
+        "common": ["-c", "--target=arm-arm-none-eabi", "-mthumb", "-Oz", "-DNDEBUG", "-flto"],
```

Comment:
No. ld flags are not taken from common.

`-Oz` is for minimizing the size. That's what the release profile does. It used to be called small after all.